contrast_security_adr/assets/logs/contrast-security-adr_tests.yaml

```
@@ -0,0 +1,57 @@
+id: "contrast-security-adr"
+tests:
+ -
+  sample: |-
+    "message":{
+      "incidentId": "INC-2025-c61",
+      "incidentName": "Path Traversal on \"/postjsoncmd\"",
+      "status": "Close",
+      "score": 9.3,
+      "severity": "Error",
+      "url": "https://apptwo.com/cs/index.html#/0f767995-4882-4c7c-889f-994d945ff0d5/incidents/INC-2025-6",
+      "createdTime": "2025-07-25T06:20:23.0000Z",
+      "summary": " observed the following suspicious value accessing the application through the HTTP Request Parameter query cat /etc/passwd",
+      "source": "security",
+      "eventType": "incidentalert",
+      "alertType": "incident_created",
+      "alertReason": "Incident Created",
+      "organizationUuid": "0f767995-4882-4c7c-889f-994d945ff0d5",
+      "relatedRules": ["path-traversal"],
+      "recommendedActions": [
+        "We did not block this attack because blocking was not enabled for DM-ADR-Web-Application in Production.",
+        "Add an exclusion for this attack event."
+      ],
+      "recommendedRunbooks": [
+        "https://dev/adr-runbooks/runbooks/path-traversal/"
+      ],
+      "timestamp": "2025-07-25T06:20:23.0000Z"
+    }
```

You should not provide the `"message":`

So it should be:
```suggestion
sample: |-
    {
      "incidentId": "INC-2025-c61",
      "incidentName": "Path Traversal on \"/postjsoncmd\"",
      "status": "Close",
      "score": 9.3,
      "severity": "Error",
      "url": "https://apptwo.com/cs/index.html#/0f767995-4882-4c7c-889f-994d945ff0d5/incidents/INC-2025-6",
      "createdTime": "2025-07-25T06:20:23.0000Z",
      "summary": " observed the following suspicious value accessing the application through the HTTP Request Parameter query cat /etc/passwd",
      "source": "security",
      "eventType": "incidentalert",
      "alertType": "incident_created",
      "alertReason": "Incident Created",
      "organizationUuid": "0f767995-4882-4c7c-889f-994d945ff0d5",
      "relatedRules": ["path-traversal"],
      "recommendedActions": [
        "We did not block this attack because blocking was not enabled for DM-ADR-Web-Application in Production.",
        "Add an exclusion for this attack event."
      ],
      "recommendedRunbooks": [
        "https://dev/adr-runbooks/runbooks/path-traversal/"
      ],
      "timestamp": "2025-07-25T06:20:23.0000Z"
    }
```